### PR TITLE
Sf7

### PR DESCRIPTION
**Pull request by dbu**
| Q             | A
| ------------- | ---
| Branch?       | "master"
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -69,5 +69,6 @@
         "allow-plugins": {
             "composer/package-versions-deprecated": true
         }
-    }
+    },
+    "minimum-stability": "dev"
```
Comment (dbu) [inline composer.json]: revert this setting before merging!

File: composer.json [hunk 2]
```diff
@@ -15,28 +15,28 @@
...
+        "symfony/form": "^6.0 || ^7.0",
         "symfony/monolog-bundle": "^3.5",
-        "doctrine/phpcr-bundle": "^2.3",
-        "symfony/serializer": "^6.0",
+        "symfony/security-bundle": "^6.0 || ^7.0",
+        "symfony/serializer": "^6.0 || ^7.0",
+        "symfony/translation": "^6.0 || ^7.0",
+        "symfony/validator": "^6.0 || ^7.0",
+        "symfony/twig-bundle": "^6.0 || ^7.0",
+        "symfony-cmf/testing": "dev-sf7 as 4.2.0",
```
Comment (dbu) [inline composer.json]: change to ^5.0.0 when https://github.com/symfony-cmf/Testing/pull/215 is merged and released

**Comment by acrobat** [comment]
Hi @dbu, is there anything I can do to help move this PR forward and have Symfony 7 support released?

**Comment by dbu** [comment]
thanks for asking. there are a bunch of dependencies i have not yet released in new versions. i hope i am getting there soon.

**Comment by acrobat** [comment]
No worries! If you need some help or I need to test anything, just let me know!

File: composer.json [hunk 3]
```diff
@@ -15,28 +15,28 @@
...
+        "symfony/form": "^6.4 || ^7.0",
         "symfony/monolog-bundle": "^3.5",
-        "doctrine/phpcr-bundle": "^2.3",
-        "symfony/serializer": "^6.0",
+        "symfony/security-bundle": "^6.4 || ^7.0",
+        "symfony/serializer": "^6.4 || ^7.0",
+        "symfony/translation": "^6.4 || ^7.0",
+        "symfony/validator": "^6.4 || ^7.0",
+        "symfony/twig-bundle": "^6.4 || ^7.0",
+        "symfony-cmf/testing": "dev-fix-test-kernel as 5.0.2",
```
Comment (dbu) [inline composer.json:39]: adjust before merging

**Comment by dbu** [comment]
will need some more testing and the slightly BC breaking cleanups for strict typing before releasing, but we are getting there  \o/

**Comment by dbu** [comment]
just realized we don't do a new major version here. so no additional strict typing in src.

@acrobat @alexander-schranz i would be glad if you can test the current 3.x branch and open issues or confirm here if it works for you, both on older projects without updating anything else, and on a symfony 7 project.

**Comment by alexander-schranz** [comment]
I did test sulu/sulu manually the routing still seems to work like expected. Over composer I can not install the current versions because `3.0.3` of symfony-cmf/routing is not tagged yet and so even `composer config minimum-stability dev` can not install it but I manually checkout `symfony-cmf/routing` and `symfony-cmf/routing-bundle` 3.x branches manually in my vendor and it seems to work like expected.

**Comment by dbu** [comment]
strange about not being able to install symfony-cmf/routing 3.0.3 - i had created the tag in february already, i can see it on packagist: https://packagist.org/packages/symfony-cmf/routing#3.0.3 (but i forgot to create the release entry in github - but packagist looks at the git tags, not at github releases)

**Comment by acrobat** [comment]
@dbu I've tested the routing-bundle 3.x-dev against symfony 6.4 and everything seems to be working so far! I'm not able to test with symfony 7 yet as I'm still waiting for other libraries to add compatiblity.

**Comment by dbu** [comment]
https://github.com/symfony-cmf/routing-bundle/releases/tag/3.1.0

**Pull request merged**